docs/fundamentals/syslib-diagnostics/syslib0004.md

```diff
@@ -19,9 +19,15 @@ The following CER-related APIs are obsolete:
 - <xref:System.Runtime.ConstrainedExecution.PrePrepareMethodAttribute?displayProperty=nameWithType>
 - <xref:System.Runtime.ConstrainedExecution.ReliabilityContractAttribute?displayProperty=nameWithType>
 
+However, the following CER-related APIs are *not* obsolete:
+
+- <xref:System.Runtime.CompilerServices.RuntimeHelpers.PrepareDelegate(System.Delegate)?displayProperty=nameWithType>
+- <xref:System.Runtime.CompilerServices.RuntimeHelpers.PrepareMethod%2A?displayProperty=nameWithType>
+- <xref:System.Runtime.ConstrainedExecution.CriticalFinalizerObject?displayProperty=nameWithType>
```

Comment:
I added this one too - please check.

Reply:
Makes sense, <https://github.com/dotnet/runtime/blob/v6.0.2/src/libraries/System.Private.CoreLib/src/System/Runtime/ConstrainedExecution/CriticalFinalizerObject.cs> does not have ObsoleteAttribute.